Libraries/LibWeb/HighResolutionTime/Performance.idl

```diff
@@ -37,8 +37,8 @@ interface Performance : EventTarget {
 
     // https://w3c.github.io/navigation-timing/#extensions-to-the-performance-interface
     // Obsolete "Navigation Timing" extensions to the Performance interface
-    [SameObject]
+    [SameObject, Exposed=Window]
```

Comment:
Theoretically - we now support `partial interface` after https://github.com/LadybirdBrowser/ladybird/pull/5559, so we could yoink these APIs to their own file and use the literal text from the spec?
```
[Exposed=Window]
partial interface Performance {
  [SameObject]
  readonly attribute PerformanceTiming timing;
  [SameObject]
  readonly attribute PerformanceNavigation navigation;
};
```

Reply:
Note that #5559 is pretty simplistic, you'd need to test whether or not `[Exposed=..]` actually gets transferred to attributes if it's defined on the partial interface level.